### ISSUE TITLE
Have to issue command twice to reveal file

### ISSUE THREAD

**Issue by kmaustral** [gr0grig/obsidian-virt-folder#8]
Thanks for your plugin. I'm beginning to use it more often. It's very useful. 

Currently, when I select Reveal File, it opens the side bar but all the folders are collapsed. It's only when I do it again that the relevant folder is shown. 

While there, it would be useful to highlight the revealed file.

**Comment by gr0grig** [comment]
Yes, such bug exists. But the opened file is already highlighted.

**Comment by kmaustral** [comment]
Thanks for the reply. The highlight doesn't show in the Primary theme. Perhaps there might be a way of customising it.

**Comment by gr0grig** [comment]
Please clarify what version of the plugin you have? The backlight did not appear immediately.

![css](https://github.com/user-attachments/assets/d9da66d7-0d59-491d-98d4-4544b090b728)

And if it's not too much trouble, can I see a screenshot like this?

**Comment by kmaustral** [comment]
I'm using 1.0.7.

I activated Reveal File for 2024.09.23

 [image: image.png]

Let me know if you have any more questions.

Thanks
Kevin

**Comment by gr0grig** [comment]
Unfortunately, when replying via email, the link to the image breaks.

**Comment by kmaustral** [comment]
Sorry. I trust this works. Nothing much to see. 
![image](https://github.com/user-attachments/assets/f2d26253-07fc-4119-82e9-43b27703be3f)

**Comment by kmaustral** [comment]
Thanks. Sorry for my ignorance, but is this code sufficient to include, or
do I need to specify the colour?

**Comment by gr0grig** [comment]
This is the original code, you need to replace the color with a suitable one from the link.

**Comment by gr0grig** [comment]
I forgot one tag. I checked it on your theme, it works.

```css
.current_note
{
    background-color: var(--background-primary) !important;
}
```

**Comment by kmaustral** [comment]
Thanks. That works now.
